README.en.md

```diff
@@ -74,12 +74,12 @@ Licensed under GPLv3+ (See LICENSE)
 
 ### Why GPL?
 
-The expected functions of Easer contain lots of tracks / captures of privacy (e.g. location, calendar), and would be able to access the Internet. We would never want a tool which is expected to better facilitate our lives to become a spying tool, so we must prevent that from happening as we can. The only way is to allow anyone to censor every part of Easer, which means to ensure Easer (and its derivated work, if any) to be opensource.  
-Because as of the design of Easer, each functionality will (in the furutre) become modules / plugins, maicious codes should also be prohibited from these parts, so we hope to rely on the feature of GPL (that derivated work should also be licensed under GPL) to enforce this.
+The expected functions of Easer require access to personal information (e.g. location, calendar) and networking capabilities. We would never want a tool that is expected to better facilitate our lives to spy on us, so we must prevent that from happening as best as we can. The only way to do this is to allow anyone to inspect every part of Easer, which is say that Easer (and any derived works) must be made open source.
```

Comment:
Personally I'd prefer "privacy".

Typo (last sentence): which is "to" say.

Reply:
Hmm. I changed it to "personal information" because it _is_ what Easer needs to access to function. Privacy is more like one's ability to keep information hidden from others and share what one wants to share. It's not the same as the information itself.

When I wrote "personal information," I was thinking of [personally identifiable information](https://en.wikipedia.org/wiki/Personally_identifiable_information). However, now that I think about it, perhaps that doesn't convey the right meaning. Would "private information" be a better choice?